kythe/go/indexer/testdata/basic/inline.go

```diff
@@ -0,0 +1,25 @@
+package subject
+
+var Primary = struct {
```

Comment:
Unless I'm missing something, you can do this instead of the manual `loc/{start,end}` goals below:

`//- @Primary defines/binding Primary`

and

`//- @MyParam defines/binding MyParam` above the relevant MyParam line below

Reply:
The reason I did not use @Primary is because @Primary would have to be defined one or more lines above var Primary. This file is testing that the changes to the indexer have successfully loaded metadata which contains specific byte location information. Since my goals are dependent on specific byte locations, adding a goal containing @Primary above var Primary would shift the byte offsets of where Primary itself occurs, hence why we used the manual loc/{start, end} syntax.